docs/src/chrome-extensions-js-python.md

```diff
@@ -82,3 +82,61 @@ def run(playwright):
 with sync_playwright() as playwright:
     run(playwright)
 ```
+
+## Testing
+
+To have the extension loaded when running tests you can use a test fixture to set the context. You can also dynamically retrieve the extension id and use it that to load and test the popup page for example.
+
+```typescript
```

Comment:
let's add an example for Python with Pytest as well? Otherwise its a block of text without a code snippet for Python users.

Reply:
I agree but I likely won't have time to write the Python example myself.

Reply:
I can follow up with that.